### ISSUE TITLE
为啥单看vae 的重构，效果完全烂掉啊？ 是不是训Diffusion Transformer 的时候，没冻住VAE 参数啊？

### ISSUE THREAD

**Issue by 353xiong** [tencent-ailab/songgeneration#67]

**Comment by LiuZH-19** [comment]
请问找到原因了嘛？ 我在对audio进行tokenizer encode和decode，音频重建的效果也很差。不知道是不是这块vae的原因？

**Comment by LiuZH-19** [comment]
能否分享一下你 audio tokenizer进行重构的代码？ 我这边存在人声错位，伴奏和人声不匹配，以及时长不对等问题。想对照下看，是否有bug。万分感谢～

**Comment by 353xiong** [comment]
> 可以试试下面的代码 def load_audio(self, f): a, fs = torchaudio.load(f) if (fs != 48000): a = torchaudio.functional.resample(a, fs, 48000) return a pmt_wav = self.load_audio(audio_path) vae_input = seperate_tokenizer.model.vae.preprocess_audio_for_encoder(pmt_wav.cuda(), 48000) vae_output = seperate_tokenizer.model.vae.encode_audio(vae_input, chunked=True) vae_output = seperate_tokenizer.model.vae.decode_audio(vae_output, chunked=True) vae_output = vae_output[0].detach().cpu() torchaudio.save(f'vae_reconstruct/{i}.flac', vae_output, 48000)

你试过啦？我测了，还是啦呀； 音频都糊成一团；
